tests/functional/job-submission/15-garbage-host-command-2.t

```diff
@@ -17,13 +17,12 @@
 #-------------------------------------------------------------------------------
 # Test recovery of a failed host select command for a group of tasks.
 . "$(dirname "$0")/test_header"
-skip_all "TODO replace this test with one checking that garbage-platform commands are handled ok"
 set_test_number 2
 
 create_test_global_config "
 [platforms]
     [[test platform]]
-        hosts = $(my-host-select)
+        hosts = localhost
```

Comment:
Debug change?

Reply:
Nope, just a mistake. I don't think that this will work at all, because we don't evaluate this.